setup.cfg

```diff
@@ -89,4 +91,12 @@ ignore = E402
 # generated
 exclude = _astropy_init.py,version.py
 
+# E402 module level import not at top of file
+# E501 line too long
+# W504 line break after operator
+# Stricter settings for people who use flake8 in their editors
+[flake8]
+ignore = E402,E501,W504
```

Comment:
add W503 instead, the new directive is to keep the operators at the beginning of the line rather at the end.

Also, I think we can have E501, with setting the line length to 100 or 120?

Reply:
Done. 120 for now.